pkg/routing/localrouter.go

```diff
@@ -172,39 +234,82 @@ func (r *LocalRouter) OnRTCMessage(callback RTCMessageCallback) {
 	r.onRTCMessage = callback
 }
 
+func (r *LocalRouter) RegisterRoomService(svc rpc.RoomServerImpl) (err error) {
+	r.roomServer, err = rpc.NewRoomServer(r.currentNode.Id, svc, r.bus)
+	return
+}
+
+func (r *LocalRouter) RegisterParticipant(roomName livekit.RoomName, identity livekit.ParticipantIdentity) error {
```

Comment:
what's calling these topic registration methods today?

Reply:
roommanager calls these during room creation/deletion and signaling session open/close similar to cloud